src/zjs_console.c

```diff
@@ -170,6 +186,10 @@ static jerry_value_t console_time(const jerry_value_t function_obj,
         return ZJS_UNDEFINED;
     }
     int sz = jerry_get_string_size(argv[0]);
+    if (sz > MAX_STR_LENGTH) {
+        ERR_PRINT("label sting too long\n");
```

Comment:
This petty comment is going to sting. (Typo!)

You'll check return value from zjs_copy_jstring instead here.